R/roxygen-examples-add-remove.R

```diff
@@ -29,7 +25,7 @@ remove_roxygen_mask <- function(text) {
 #' #' @examples c(1, 2)
 #' @keywords internal
 remove_roxygen_header <- function(text) {
-  gsub("^[\\s\t]*@examples(If)?(\\s|\t)*", "", text, perl = TRUE)
+  gsub(R"(^[\s\t]*@examples(If)?(\s|\t)*)", "", text, perl = TRUE)
```

Comment:
Was this a bug? That is, should this actually have been `\\s|\\t`?

Reply:
`\s` stands for space, no? So it should capture spaces and tabs.

Reply:
@IndrajeetPatil are you confident in these topics?